extract-racer-tab.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import sys
+
+strings = {}
+
+for path in sys.argv[1:]:
+  with open(path, 'rb') as in_file:
+    data = in_file.read()
+
+    scanlist = [
+      b"/LANGID",
+      b"/CREDITS_H_",
+      b"/MONDOTEXT_H_",
+      b"/SCREENTEXT_"
+    ]
+
+    for s in scanlist:
+      p = 0
```

Comment:
Should use same variable names as in the other script. These names are confusing..